components/common/Table.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect } from 'react'
+import Tooltip from './Tooltip'
+import Link from '../Link'
+import { useTable, useSortBy } from 'react-table'
+import { toPairs } from 'ramda'
+import { formatDecimal } from '../../lib/utils/format'
+const glossary = require('../../lib/i18n/glossary_en.json')
+const formattedNum = ({ cell: { value } }) => formatDecimal(value)
+
+function chooseRenderer (datatype, idMap) {
```

Comment:
I would probably rename this to something like selectCellFormater or something else a little more specific